packages/client/src/fm-player/services/connected-users-service.ts

```diff
@@ -0,0 +1,24 @@
+import mitt, { Emitter, Handler } from 'mitt';
+
+const emitter: Emitter = mitt();
+
+let connectedUsers: string[] = [];
+
+function getUsers(): string[] {
+  return connectedUsers;
```

Comment:
I feel like this function should return a copy of that array, so that other parts of the application would not be able to mutate it's global state. 

Essentially make this array be only internally mutable from this file.

Same thing down below in the event emitter.

Reply:
But I do admit that this is my inner ~demons~ Rust-isms showing up.

Reply:
I agree, I added a fix